src/components/Form/ProposalForm.tsx

```diff
@@ -120,27 +117,24 @@ function ProposalForm({
           values.rateToken,
           token.decimals,
         );
+        console.log({parsedRateAmount})
         const now = Math.floor(Date.now() / 1000);
         const convertExpirationDate = now + 60 * 60 * 24 * values.expirationDate;
         const convertExpirationDateString = convertExpirationDate.toString();
 
         const parsedRateAmountString = parsedRateAmount.toString();
 
-        const cid = await postToIPFS(
-          JSON.stringify({
-            about: values.about,
-            video_url: values.video_url,
-          }),
-        );
+        
+        const propsalMeta = {
+          about: values.about,
+          video_url: values.video_url,
+          test: "hello"
```

Comment:
why this test ?

Reply:
I added this for testing 🤦 

Does not need to be part of this PR